src/EC2Store.js

```diff
@@ -22,7 +22,59 @@ class EC2Store {
 			let ec2 = new AWS.EC2();
 			ec2.describeSnapshots({}, function (err, response) {
 				if (err) reject(err);
-				else resolve(response);
+
+				else {
+					var taggedForBackup = response.Snapshots.filter(function(obj){
+						for (var i=0; i<obj.Tags.length; i++) {
+							if (obj.Tags[i].Key === 'backups:config-v0') {
+								return true;
+							}
+						}
+						return false;
+					});
+
+					resolve(taggedForBackup.map(function(obj){
+
+						var filteredForName = obj.Tags.filter(function(obj){
+							if (obj.Key === 'Name'){
```

Comment:
While this `obj` is a tag object. Also note that it is hard to tell if you are referring to the Tag `obj` or the Snapshot `obj` from the lines above.